book/src/ledger-replication.md

```diff
@@ -29,13 +28,12 @@ core. The total space required for verification is `1_ledger_segment +
 ## Network
 
 Validators for PoRep are the same validators that are verifying transactions.
-They have some stake that they have put up as collateral that ensures that
-their work is honest. If you can prove that a validator verified a fake PoRep,
-then the validator will not receive a reward for that storage epoch.
+If a replicator can prove that a validator verified a fake PoRep, then the
+validator will not receive a reward for that storage epoch.
```

Comment:
This is a really weak disincentive.  Validator should be slashed or frozen.  But we need to have a robust slashing design somewhere.

Reply:
Haha i just pulled the slashing thing out of it because it's so hand wavy.

Reply:
To be fair, we collect fees from those validator submissions so it's not entirely unpunished.